semcore/date-picker/src/MonthRangePicker.jsx

```diff
@@ -2,10 +2,12 @@ import React from 'react';
 import dayjs from 'dayjs';
 import createComponent from '@semcore/core';
 import shortDateRangeFormat from './utils/shortDateRangeFormat';
-import { Header, Next, Period, Popper, Prev, Title, Trigger } from './components';
+import { Header, Next, Period, Popper, Prev, Title, Trigger, InputTrigger } from './components';
 import { CalendarMonths as Calendar } from './components/Calendar';
 import RangePickerAbstract, { Apply, Reset } from './components/RangePickerAbstract';
 
+const inputTriggerParts = { day: false, month: true, year: true };
```

Comment:
why there is a day

Reply:
Why not?

Reply:
Days are disabled (`day: false`) 🙂

Reply:
But I can see them in the screenshot above